app/jenkins-for-jira-ui/src/components/MainPage/TopPanel/TopPanel.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { TopPanel } from './TopPanel';
+
+describe('TopPanel Component', () => {
```

Comment:
Not sure these add much value? Maybe I just omit these? 🤔

Reply:
Yeah this is just testing react is working as expected. It'd be good to update these to user flow test cases instead when you build the functionality

Reply:
I'll rip em out and add something more substantial when we have functionality.